command/rootchain/deploy/deploy.go

```diff
@@ -546,18 +549,18 @@ func deployContracts(outputter command.OutputFormatter, client *jsonrpc.Client,
 			}
```

Comment:
Be careful here, IMO we should add results into the `commandResults` slice. 

Also on line 542, we should probably change that to `MessageResult` and add it to `commandResults` slice as well, instead of printing it straight away to `outputter`.